.github/files/multi-platform.Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM alpine
+FROM alpine:3.21
```

Comment:
we have automatic dependency updates on
=> we get notified if something here is breaking => pinning to minor is imo fine for this one ^^